sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -128,7 +129,7 @@ protected DigitalTwinsClient()
         /// <returns>The deserialized application/json digital twin and the HTTP response <see cref="Response{T}"/>.</returns>
         /// <typeparam name="T">The type to deserialize the digital twin to.</typeparam>
         /// <exception cref="RequestFailedException">
-        /// The exception that captures the errors from the service. Check the <see cref="RequestFailedException.ErrorCode"/> and <see cref="RequestFailedException.Status"/> properties for more details.
+        /// The exception that captures the errors from the service. Check the <see cref="RequestFailedException.ErrorCode"/> and <see cref="MessageFailedException.Status"/> properties for more details.
```

Comment:
I understand that this is an acceptable change according to https://learning.oreilly.com/library/view/framework-design-guidelines/9780135896457/app04.xhtml#secD_8

**D.8.1 Moving Members to a Base Class**
Moving a member to a base class is gracefully handled by the .NET CLR and is generally supported for both static and instance members in .NET languages.

The .NET BCL team considers this an acceptable change, but would warn you that it cannot be safely undone.